pages/_document.tsx

```diff
@@ -1,9 +1,22 @@
 import { Html, Head, Main, NextScript } from 'next/document';
+import Script from 'next/script';
 
 export default function Document() {
   return (
     <Html>
       <Head>
+        <Script
```

Comment:
Can you move this to `_app.tsx` instead please? I have an example in my PR here: #20, you can copy it from there if you would like.

Reply:
@jlf0dev I've added GTM script to _app.tsx